openedx/core/djangoapps/ace_common/template_context.py

```diff
@@ -8,6 +8,7 @@
 
 from edxmako.shortcuts import marketing_link
 from openedx.core.djangoapps.theming.helpers import get_config_value_from_site_or_settings
+from openedx.core.djangoapps.site_configuration.helpers import get_current_site_configuration_values
```

Comment:
@shimulch, this appears to be an unused import

Reply:
Thanks for spotting this, removed.